test/built-ins/Math/E/S15.8.1.1_A1.js

```diff
@@ -2,15 +2,12 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-info: Math.E is approximately 2.7182818284590452354
+info: Math.E is 2.7182818284590452354
 es5id: 15.8.1.1_A1
 description: Comparing Math.E with 2.7182818284590452354
-includes:
-    - math_precision.js
-    - math_isequal.js
 ---*/
 
 // CHECK#1
-if (!isEqual(Math.E, 2.7182818284590452354)) {
-  $ERROR('#1: \'Math.E is not approximately equal to 2.7182818284590452354\'');
+if (Math.E !== 2.718281828459045) {
```

Comment:
further checks: info and description values are valid with the specs. Although, `2.718281828459045` is used in V8, SpiderMonkey, and Safari. I didn't check Edge.

Should we update the specs to reflect reality or fix these tests?